tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -1584,7 +1584,11 @@
           <xsl:value-of select="$typename" />
           <xsl:text>",</xsl:text>
         </xsl:if>
-        <xsl:text>"type":"object","properties":{"results":{</xsl:text>
+        <xsl:text>"type":"object","properties":{</xsl:text>
+        <xsl:if test="$inResponse">
+          <xsl:text>"__count":{"type":"string","description":"The number of entities in the collection. Available when using the [$inlinecount](https://help.sap.com/doc/5890d27be418427993fafa6722cdc03b/Cloud/en-US/OdataV2.pdf#page=67) query option."},</xsl:text>

```

Comment:
Please create a reuse structure next to `definitions/error` and use `{"$ref":"#/definitions/count"}` here

Reply:
> And the count can be a number or a string.
You sure about that? I've only ever seen __count as a string both in "the wild" and in documentation

Reply:
You‘re right, in V2 it is always a string, only in V4 it depends on the requested number representation.